Sources/_StringProcessing/Regex/DSLTree.swift

```diff
@@ -164,6 +164,96 @@ extension DSLTree {
       indirect case subtraction(CustomCharacterClass, CustomCharacterClass)
       indirect case symmetricDifference(CustomCharacterClass, CustomCharacterClass)
     }
+    
+    public struct AsciiBitset {
+      let isInverted: Bool
+      let isCaseInsensitive: Bool
+      var a: UInt64 = 0
+      var b: UInt64 = 0
```

Comment:
This is ok, another option is something like `var rawValue: (UInt64, UInt64)` and add the OptionSet conformance.

Reply:
`(UInt64, UInt64)` doesn't have `FixedWidthInteger` conformance so I think we'd end up having to implement all of the set operations ourselves. I just left it as is